sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -916,7 +916,7 @@ object RapidsConf {
         |On startup use: `--conf [conf key]=[conf value]`. For example:
         |
         |```
-        |${SPARK_HOME}/bin/spark --jars 'rapids-4-spark_2.12-0.4.0.jar,cudf-0.18-cuda10-1.jar' \
+        |${SPARK_HOME}/bin/spark --jars 'rapids-4-spark_2.12-0.4.0.jar,cudf-0.18.1-cuda10-1.jar' \
```

Comment:
This change will make the configs.md changed after mvn verify, I think.
It'll cause the premerge failure since Peixin enabled the check of any changes after 'mvn verify'.
Do we really need to update it in this PR?

Reply:
Ok, let me check it

As the scala file will be built into the release binary. If the PR changes  related docs, I think we should also update the docs